main.py

```diff
@@ -53,6 +56,30 @@ async def websocket_endpoint(websocket: WebSocket):
             break
 
 
+@app.get("/metadata")
+def return_features(user_agent: Union[str, None] = Header(default=None)):
+    """Handle metadata for OS compatilibity"""
+    if "windows" in user_agent.lower():
```

Comment:
Instead of user agent operating system, i.e. the browser's operating system, we need to decide features based on the server's operating system. For example, `hscroll` is not supported when remote-play server is running on Windows. We can use python's `sys.platform` to check this.

Reply:
sounds good,